variables.tf

```
@@ -175,6 +175,12 @@ variable "replication_group_id" {
   default     = ""
 }
 
+variable "snapshot_arns" {
+  type        = list(string)
+  description = "(Optional) A single-element string list containing an Amazon Resource Name (ARN) of a Redis RDB snapshot file stored in Amazon S3. Example: arn:aws:s3:::my_bucket/snapshot1.rdb"
```

```suggestion
  description = "A single-element string list containing an Amazon Resource Name (ARN) of a Redis RDB snapshot file stored in Amazon S3. Example: arn:aws:s3:::my_bucket/snapshot1.rdb"
```

for consistency with other variables